src/components/Modal/VoteProposalModal.tsx

```diff
@@ -1,12 +1,33 @@
-import { Button } from '@/components/Button'
 import { Modal } from '@/components/Modal/Modal'
-import { Input } from '@/components/Input'
-import { Header, Label, Paragraph, Span, Typography } from '@/components/Typography'
-import { formatNumberWithCommas, shortAddress, toFixed, truncateMiddle } from '@/lib/utils'
+import { Typography } from '@/components/Typography'
+import { formatNumberWithCommas, shortAddress, truncateMiddle } from '@/lib/utils'
 import { FC, useState } from 'react'
-import { FaCopy } from 'react-icons/fa6'
+import { FaCopy } from 'react-icons/fa'
 import { Address } from 'viem'
-import Image from 'next/image'
+import { Button } from '@/components/Button'
+import { cn } from '@/lib/utils'
+import { round } from '@/lib/big'
+
+const QuestionMarkWithTooltip = () => (
```

Comment:
For the Question mark with tooltip, please take a look at the current implementation that we already have. For example in `app/proposals/page.tsx`. We have a dedicated Popover component and a  `QuestionIcon`.

Reply:
Awesome. Done, thanks for pointing that out